src/functions/api.js

```diff
@@ -10,11 +10,11 @@ export function getReviews(categoryName) {
       return res.data.reviews;
     });
 }
-export function getReviewByID(ID) {
+export function getReviewByID(review_id) {
   return axios
-    .get(`https://kats-nc-games.herokuapp.com/api/reviews/${ID}`)
+    .get(`https://kats-nc-games.herokuapp.com/api/reviews/24`)
```

Comment:
id 24 is hardcoded in here causing the review with ID 24 'escape the dark sector' to be rendered no matter which review is clicked